core/util.py

```diff
@@ -65,3 +65,17 @@ def get_project_number(project_id):
     :return: project number
     """
     return int(project_id.split("-")[2])
+
+
+def construct_index(section: str, column: str, rows: list[int]) -> str:
+    """Constructs the index of an error from the column and rows it occurred in.
+
+    :param section: the internal table name where the error occurred
+    :param column: the internal table name where the error occurred
+    :param rows: tuple or int of row indexes where the error occurred
+    :return: indexes tuple of constructed letter and number indexes
+    """
+
+    column_letter = TABLE_AND_COLUMN_TO_ORIGINAL_COLUMN_LETTER[section][column]
+    indexes = ", ".join([column_letter.format(i=row + 2) for row in rows])
```

Comment:
Could the reason behind this increment of 2 be documented in the docstring and/or a comment please?